terracumber_config/tf_files/SUSEManager-Tests-Reporting-Hub.tf

```diff
@@ -0,0 +1,221 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = "string"
+  default = "https://ci.suse.de/view/Manager/view/Manager-Test/job/manager-TEST-reportdb-hub-tests"
+}
+
+variable "CUCUMBER_COMMAND" {
+  type = "string"
+  default = "export PRODUCT='Uyuni' && run-testsuite #TODO run only sanity, core and hub tests, including reporting"
+}
+
+variable "CUCUMBER_GITREPO" {
+  type = "string"
+  default = "https://github.com/uyuni-project/uyuni.git"
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = "string"
+  default = "master"
```

Comment:
If you have a concrete branch where you are doing your development, you can always pass it here.
So instead of testing master, you test your branch. Same for the repository in the variable above.